packages/components/psammead-numbered-list/src/index.jsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
```

Comment:
We've switched styled-components for Emotion

The rest of your code does seem compatible with this change - just need to update the import 🙂 
```suggestion
import styled from '@emotion/styled';
```